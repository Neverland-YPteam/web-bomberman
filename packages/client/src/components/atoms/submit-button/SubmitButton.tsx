import React from 'react'
import { Button } from '@mui/material'

interface Props {
  children: string;
}

const SubmitButton = ({ children }: Props) => {
  return (
    <Button
      type="submit"
      fullWidth
      variant="contained"
      sx={{
        mt: 3,
        mb: 2,
        bgcolor: ({palette}) => palette.info.dark,
        '&:hover': {
          bgcolor: ({palette}) => palette.info.main
        }
      }}
    >
      {children}
    </Button>
  )
}

export default SubmitButton
