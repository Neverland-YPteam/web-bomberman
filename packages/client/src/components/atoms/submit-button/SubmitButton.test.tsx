import React from 'react'
import { create } from 'react-test-renderer'
import { SubmitButton } from '@atoms/submit-button/index'


it('SubmitButton snapshot', () => {
  const actualSnapshot = create(<SubmitButton>Text</SubmitButton>)
    .toJSON()

  expect(actualSnapshot).toMatchSnapshot()
})
