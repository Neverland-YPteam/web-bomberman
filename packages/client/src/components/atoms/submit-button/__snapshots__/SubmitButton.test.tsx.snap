// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubmitButton snapshot 1`] = `
<button
  className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth css-19mc8ns-MuiButtonBase-root-MuiButton-root"
  disabled={false}
  onBlur={[Function]}
  onContextMenu={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="submit"
>
  Text
</button>
`;
